mono/native/Makefile.am

```diff
@@ -34,7 +34,9 @@ common_sources = \
 	../../external/corefx/src/Native/Unix/System.Native/pal_uid.c \
 	../../external/corefx/src/Native/Unix/System.Native/pal_uid.h \
 	../../external/corefx/src/Native/Unix/System.Native/pal_time.c \
-	../../external/corefx/src/Native/Unix/System.Native/pal_time.h
+	../../external/corefx/src/Native/Unix/System.Native/pal_time.h \
```

Comment:
We should not load existing Mono.Native at all, why is it needed?

Reply:
@vargaz @marek-safar I didn't update the native library loader to use the paths from the coreclr_initialize property list yet - so right now Mono is still trying to resolve .dylib loads against $(prefix), not `NATIVE_DLL_SEARCH_DIRECTORIES` or `APP_NI_PATHS`  (see https://github.com/mono/mono/blob/55d29aa7d4038846b12ce1b9350ccab738b48a2f/mono/mini/main-core.c#L120-L123)